src/SFA.DAS.LoginService.Application/Invitations/CreateInvitation/CreateInvitationHandler.cs

```diff
@@ -55,8 +55,8 @@ public class CreateInvitationHandler : IRequestHandler<CreateInvitationRequest,
             }
             
             _logger.LogInformation($"CreateInvitationHandler : Client: {JsonConvert.SerializeObject(client)}");
-            
-            var existingUser = await _userService.FindByEmail(request.Email);
+
+            var existingUser = await _userAccountService.FindByEmail(request.Email);
```

Comment:
I'm curious about this change - does the `UserManager` (the backing behind the previous `UserService`) not contain the user?

Reply:
UserManager contains HttpContext and isn't available with a message handler. So this rework allows us to call this application layer from the web or the MessageHandler